.travis.yml

```diff
@@ -10,7 +10,7 @@ before_script:
   - go get -u github.com/mattn/goveralls
 
 script:
-  - make lint
+  - make test
```

Comment:
why? make test and make test-coverage are the same

Reply:
make test will invoke make lint, if in the future we add something else to test we would not need to update the travis file.

while make test coverage indirectly invokes `go test ../.` its implementation detail imo(which could change)

logically when looking at the travis file we understand that it runs tests, then calculates coverage and publishes to coveralls.

test-coverage is not invoked by make test today